src/aliceVision/CMakeLists.txt

```
@@ -49,8 +49,7 @@ endif()
 if(ALICEVISION_BUILD_MVS)
   add_subdirectory(lightingEstimation)
   add_subdirectory(mesh)
-  add_subdirectory(mvsData)
-  add_subdirectory(mvsUtils)
+    add_subdirectory(mvsUtils)
```

i think it should be:
```suggestion
    add_subdirectory(mvsData)
    add_subdirectory(mvsUtils)
```
https://github.com/alicevision/AliceVision/blob/develop/src/aliceVision/CMakeLists.txt#L48

---

@fabiencastan suggested above 
> This creates the CI failure.
> You need to remove the duplicated line (line 52), to avoid having the same `add_subdirectory` twice.
> So keep this line, but remove the one inside `if(ALICEVISION_BUILD_MVS)`.

so to have the code as 
```
if(ALICEVISION_BUILD_HDR)
  add_subdirectory(hdr)
endif()

if(ALICEVISION_BUILD_HDR OR ALICEVISION_BUILD_MVS)
  add_subdirectory(mvsData)
endif()

# MVS modules

if(ALICEVISION_BUILD_MVS)
  add_subdirectory(lightingEstimation)
  add_subdirectory(mesh)
    add_subdirectory(mvsUtils)
  add_subdirectory(fuseCut)
```
I think the issue is using `mvsData` twice. Your suggestion is good, but that would mean we would have to remove 
```
if(ALICEVISION_BUILD_HDR OR ALICEVISION_BUILD_MVS)
  add_subdirectory(mvsData)
endif()
```
Would that be OK to do?